plugins/ltac/extratactics.mlg

```diff
@@ -36,20 +36,30 @@ TACTIC EXTEND assert_succeeds
 END
 
 TACTIC EXTEND replace
```

Comment:
why do we have so many TACTIC EXTEND blocks here instead of a single one with all the replace variants?

Reply:
What exactly is the fear about?

Reply:
Things may randomly stop parsing if you factorize the rules. I guess one can give it a try in another PR, in any case this one is following the same pattern as before.